tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+You will also need to update: 
+* the "version" column with the new version
+* the "Full Example" column with the link pointing to examples for this new version, see more on the [Update examples](#6-update-examples) section
+
+You should update the column "Official Type" if and only if you modified the main parent type from Schema.org. For instance, in previous versions the Bioschemas TrainingMaterial profile had CreativeWork as the main parent Schema.org type but it was changed to LearningResource from version 0.8
+
+### 2.2. The Schema.org mapping tab
+
+Here you will make the changes to the profile. There are a couple of possibilities here.
+* You want to update a property that already exists (e.g., there is typo, description should be updated, a new controlled vocabulary will be added, a new example is necessary, expected Schema.org types have changed, etc.)
+  * Go to the column you want to change and update it as needed
+* You want to add a new property
+  * If it is a property belonging to a parent type already recorded, see Figure 1 below. Our ScholarlyArticle example already extends ScholarlyArticle, Article, CreativeWork and Thing
+    * Add a new column under the corresponding type. Let's suppose you want to add the property "wordCount" that comes from the type Article
+    * Go to the parent type in Schema.org, in this case [Article](https://schema.org/Article) and copy the property name, the expected types and the description and paste them on the first three columns of the row you just created
+    * Fill in the rest of the columns as needed (remember, columns "Type" and "Type URL" should be left empty)
+  * If it is a property belonging to a new parent
+    * Add a new row indicating what type is being extended
+    * Follow the process as described for properties belonging to a parent already recorded
+
+| ![Figure 1. From Schema.org to profile spredsheet](/tutorials/images/from_schema_to_spreadsheet.png) |
+| __Figure 1. From Schema.org to profile spredsheet__ |
+
+### 2.3. Publish the Bioschemas fields to the Web
+
+Once you have finished with the changes on the spreadsheet, go to File/Publish to the web/, select "Bioschemas Fields" and "CSV", publish and copy the link, see Figure 2 below.
+
+| ![Figure 2. Publish Bioschemas fields to the web](/tutorials/images/publish-to-web.png) |
+| __Figure 2. Publish Bioschemas fields to the web__ |
+
+
+## 3. Get the code for the new draft profile
+
+Now you are ready to the the machine-processable Bioshemas version of your new draft profile.
+
+* Go to [GOweb repository](https://github.com/BioSchemas/bioschemas-goweb) and get the executable version corresponding to your Operational System. 
+* Run GOWeb using the CSV file you created on the previous section [Publish the Bioschemas fields to the Web](#23-publish-the-bioschemas-fields-to-the-web)
+* A YAML file will be created, you will use that one on the next section [Update the website](#4-update-the-website)
+
+## 4. Update the website
```

Comment:
I would suggest breaking this up into more subsections and using less bullets. The code blocks don't interact well with the bullet lists.

Reply:
What subsections do you suggest?

Reply:
Perhaps sub headings for:
- Create a new version page
- Update the redirects
- Include the GOWeb output
- Update profile version

_Not sure what the last heading should be_

Reply:
Ah! the comment was about the website section, it all makes sense now, sorry I saw the chunk of test and thought it was about the GOweb part. 
Subsections added.